examples/tv-app/android/include/application-launcher/ApplicationLauncherManager.cpp

```diff
@@ -43,41 +43,39 @@ CHIP_ERROR ApplicationLauncherManager::proxyGetApplicationList(chip::app::Attrib
     });
 }
 
-ApplicationLauncherResponse applicationLauncherClusterLaunchApp(chip::EndpointId endpoint, ApplicationLauncherApp application,
+ApplicationLauncherResponse applicationLauncherClusterLaunchApp(chip::EndpointId endpoint, Application application,
```

Comment:
These methods will be moved to the delegates in the next PR. Same as for Content Launcher PR:
https://github.com/project-chip/connectedhomeip/pull/12961

So there's no need to waste too much time on this as it will be soon removed.